docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe.md

```diff
@@ -134,3 +134,96 @@ console data:import glossary
+
+You can also set up a cronjob that triggers the event for the transition when you have more sophisticated requirements e.g. transfer money to merchants every last friday of the month.
+
+
+### App Configuration
+
+The Stripe App is capable of handling payments in a Marketplace context. This can be configured in the AppStore Catalog by selecting the business model "Marketplace". When selecting the Marketplace business model, you need to pass the required configuration values for:
+- Stripe Account ID 
+- Stripe Publishable Key
+- Stripe Secret Key
+
+When you save the configuration an asynchronous message will be sent to your application to get details for Merchants on how to onboard to the Stripe App. The onboarding is required to let Stripe know which Merchants are part of the Marketplace and to handle the payments, transfer money, and reverse transfers accordingly.
+
+From this moment your customers can use Stripe to pay for their orders.
+
+### Merchant Onboarding Process
+
+To be able to transfer money from the Marketplace Account to the Merchants, Merchants need to be onboarded to the Stripe App. When you configured the Stripe App in Marketplace mode, the Merchant onboarding process is initiated. The process is as follows:
+
+- Merchants need login to the Merchant Portal.
+- Merchants need to open the "Payment Settings" section.
+- On the "Payment Settings" the Merchant will see a new section "Stripe Onboarding".
+- The Merchant needs to click on the "Connect with Stripe" button.
+- An onboarding page of Stripe will be opened and the Merchant needs to fill in his data.
+- After the onboarding is completed, the Merchant will be redirected back to the Merchant Portal.
+
+There are several states the onboarding process can be in:
+- **Pending**: The Merchant started the onboarding process but did not complete it.
+- **Enabled**: The Merchant completed the onboarding process and is ready to receive payments.
+- **Restricted**: Additional data from the Merchant are required. When not updated the payouts will be paused after some period of time.
+- **Restricted Soon**: Additional data from the Merchant are required. When not updated the payouts will be paused in the near future.
+- **Pending**: The Merchant onboarding is not completed and must be finalized.
+- **Rejected**: The Merchant onboarding was rejected and need to contact you to clarify the issue.
+
+**Note**: You need to inform your Merchants about the required onboarding so they know about the process. Spryker doesn't support this as this is different for each project.
+**Note**: The onboarding process is handled by Stripe. You can see the status of the onboarding in the Stripe Dashboard.
+
```

Comment:
I think it's also suitable for the Configuration page or even for Stripe app description page `/docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/stripe.md`
@andriitserkovnyi  what do you think?